cmd/proxy/main.go

```diff
@@ -2,19 +2,27 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
+	"os"
 	"path/filepath"
 
 	"github.com/gomods/athens/cmd/proxy/actions"
+	"github.com/gomods/athens/pkg/build"
 	"github.com/gomods/athens/pkg/config"
 )
 
 var (
 	configFile = flag.String("config_file", filepath.Join("..", "..", "config.dev.toml"), "The path to the config file")
+	version    = flag.Bool("version", true, "Print version information and exit")
```

Comment:
I think the default value should be false otherwise if you run the proxy without any flags it will print the version and exit.

Reply:
That's probably why the ci is timing out to lol